OJP/OJP_JourneySupport.xsd

```diff
@@ -63,6 +63,11 @@
 		</xs:annotation>
 		<xs:sequence>
 			<xs:element ref="JourneyRef"/>
+			<xs:element name="PublicCode" type="xs:normalizedString" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Public code of the SERVICE JOURNEY.</xs:documentation>
```

Comment:
As the element name "Public Code" has been chosen to be in line with NeTEx as @ue71603 wrote this should be reflected in the annotation like done in other elements, e.g.

```
<xs:element name="PublishedServiceName" type="InternationalTextStructure">
	<xs:annotation>
		<xs:documentation>Line name or service description as known to the public, f.e. "512", "S8" or "Circle Line" or "ICE 488".</xs:documentation>
	</xs:annotation>
</xs:element>
<xs:element name="TrainNumber" type="xs:normalizedString" minOccurs="0">
	<xs:annotation>
		<xs:documentation>Contains the TrainNumber description from NeTEx. If several TrainNumber types exist, use the commercial number.</xs:documentation>
	</xs:annotation>
</xs:element>
```